src/main/java/teammates/sqllogic/core/UsersLogic.java

```diff
@@ -138,6 +146,23 @@ public List<Instructor> getInstructorsForCourse(String courseId) {
         return instructorReturnList;
     }
 
+    /**
+     * Check if the instructors with the provided emails exist in the course.
+     */
+    public boolean verifyInstructorsExistInCourse(String courseId, List<String> emails) {
+        List<Instructor> instructors = usersDb.getInstructorsForEmail(courseId, emails);
```

Comment:
maybe instead of returning a list, we should build a hashmap for this. Otherwise we have to iterate through the whole list to check existence